supertokens_python/recipe/session/asyncio/__init__.py

```diff
@@ -14,7 +14,6 @@
 from typing import Any, Dict, List, Union, TypeVar, Callable, Optional
 
 from supertokens_python.exceptions import SuperTokensError
-from supertokens_python.framework.request import BaseRequest
 from supertokens_python.recipe.openid.interfaces import (
     GetOpenIdDiscoveryConfigurationResult,
```

Comment:
need to make sure that the relevant changes are done in syncio as well. For example `get_session_without_request_response` is not there in syncio.